Meadowlark-js/packages/meadowlark-authz-server/src/handler/RequestToken.ts

```diff
@@ -4,42 +4,34 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 import querystring from 'node:querystring';
-import memoize from 'fast-memoize';
 import { create as createJwt } from 'njwt';
 import { Logger } from '@edfi/meadowlark-core';
+import { signingKey } from '../model/SigningKey';
 import { admin1, client1, client2, client3, client4 } from '../security/HardcodedCredential';
 import type { Jwt } from '../security/Jwt';
 import type { AuthorizationResponse } from './AuthorizationResponse';
 import type { AuthorizationRequest } from './AuthorizationRequest';
 import type { RequestTokenBody } from '../model/RequestTokenBody';
-import type { BodyValidation } from '../validation/BodyValidation';
-import { validateRequestTokenBody } from '../validation/ValidateRequestTokenBody';
+import { BodyValidation, validateRequestTokenBody } from '../validation/BodyValidation';
 import type { GetAuthorizationClientResult } from '../message/GetAuthorizationClientResult';
 import { ensurePluginsLoaded, getAuthorizationStore } from '../plugin/AuthorizationPluginLoader';
 import { hashClientSecretHexString } from '../security/HashClientSecret';
+import { extractAuthorizationHeader } from './AuthorizationHeader';
+import { TOKEN_ISSUER } from '../security/TokenIssuer';
 
 const moduleName = 'RequestToken';
 
 type ParsedRequestTokenBody =
   | { isValid: true; requestTokenBody: RequestTokenBody }
   | { isValid: false; failureMessage: string };
 
-function signingKey(): Buffer {
-  const signingKeyEncoded = process.env.SIGNING_KEY;
-  if (signingKeyEncoded == null) {
-    throw new Error('Must have a base-64 encoded signing key. Try creating a new one with `yarn createKey`');
-  }
-  return Buffer.from(signingKeyEncoded, 'base64');
-}
-const cachedSigningKey = memoize(signingKey);
-
 /*
  * Creates a standard Meadowlark Jwt.
  */
 function createToken(clientId: string, vendor: string, roles: string[]): Jwt {
-  const claims = { iss: 'ed-fi-meadowlark', aud: 'meadowlark', roles, client_id: clientId };
+  const claims = { iss: TOKEN_ISSUER, aud: TOKEN_ISSUER, roles, client_id: clientId };
```

Comment:
These will be environment variables in a future ticket, FYI. Easy enough to rename when we get there - so no need to modify right now.